### DISCUSSION TITLE
(Reopened) Tag Editor gone?

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#619] [Fixed Issues]
I can't find the tag editor anymore (rAudio 1 • 20211114)?

**Comment by phreatic62** [comment]
Still there for me, but I have to be in the 'Browse Directory' mode. 
This has always been the case, but it caught me out at first.

**Comment by klassikmann** [comment]
Interesting + thank you.
Never Seen the 'Browse Directory' mode before. But alas: I close the ticket.

**Comment by cmh714** [comment]
ok, I admit I feel like an idiot......what tag editor? and where is Browse Directory? dont think I have ever seen either of those....TIA

**Reply by klassikmann** [reply]
With the tag editor you are able to display and/or edit the tag values (title, artist, composer, genre, ...) of an album or of a single track (track title, track number, ...). In the past the menu item "Tag editor" was present in the library view after selecting an album. Nowadays this menu item will be available in the album mode only after selecting first a coverart and second "Browse directory". 

My point of view: The current implementation is too limited. Personally I have no problems because I perform all tag editing offline out of rAudio. After reconsidering the situation I decided to re-open the ticket.

**Reply by rern** [reply]
I've found the Tag Editor might have been missing. To be fixed in next update.

![0](https://user-images.githubusercontent.com/10094236/141888163-ed953c3c-8fff-474e-9010-378b58e0ed48.jpg) 

![1](https://user-images.githubusercontent.com/10094236/141888170-a60a41c8-a112-4e4b-9f03-af03aa0c00b8.jpg)

@klassikmann,

Metadata tags are directory/file based. 
Editing tags while in modes other than that would cause unexpected results for tracks which exist in multiple directories or compilation albums.

**Comment by klassikmann** [comment]
Fixed in rAudio 1 • 20211121
